siac/hostdbcmd.go

```diff
@@ -117,15 +120,30 @@ func hostdbcmd() {
 				uptimeRatio = float64(uptime) / float64(uptime+downtime)
 			}
 
+			scanHistStr := ""
+			displayScans := host.ScanHistory
+			if len(host.ScanHistory) > scanHistoryLen {
+				displayScans = host.ScanHistory[len(host.ScanHistory)-scanHistoryLen:]
+			}
+			for _, scan := range displayScans {
+				if scan.Success {
+					scanHistStr += "1"
+				} else {
+					scanHistStr += "0"
+				}
+			}
```

Comment:
I prefer the 0's and 1's. I think they make it pretty easy to see what is happening.

Generally it's not random, it's either a bunch of '1s' with the occasional streak of 0's or it's just a bunch of 0's.